semgrep/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awaremonorepo.yaml-dependency_awaremonorepo/results.json

```diff
@@ -14,12 +14,12 @@
     {
       "check_id": "rules.dependency_aware.js-sca",
       "end": {
-        "col": 6,
-        "line": 1,
-        "offset": 5
+        "col": 0,
```

Comment:
Ok yeah this test is definitely non-determenistic and bad. It's being ignored by CI right now so nothing is breaking. Going to figure out why this is happening when I'm back from vacation.